tests/init_test.py

```diff
@@ -39,6 +39,88 @@ def test_should_only_send_color_xy_if_both_color_xy_and_color_temperature_are_gi
         self.assertNotIn('color_y', sent_data['desired_state'])
 
 
+class PowerStripStateTests(unittest.TestCase):
+
+    def test_should_show_powered_state_as_false_if_device_is_disconnected(self):
+        response = """
+        {
+            "data": {
```

Comment:
I would have expected "``` data ```" to have been a list of dicts, not just a single dict.

Is this really the format of the response you got back from the Wink API?  If so, do you only have one device connected to your hub?

If this is actually the format, we'll have to update ``` get_devices_from_response_dict ``` to support "``` data ```" as *either* a ``` list ``` or a ``` dict ```.